### ISSUE TITLE
Can't access inria gitlab to access Docker file

### ISSUE THREAD

**Issue by vnckppl** [aramis-lab/clinica#416]
In [this](https://github.com/aramis-lab/clinica/blob/dev/docs/BeforeYouInstall.md) documentation file there is a link to [this gitlab](https://gitlab.inria.fr/users/sign_in) that is unaccessable for outsiders. Is there another link to a Docker recipe file?

**Comment by ghisvail** [comment]
I don't think we are ready to share an official recipe publicly yet. The one hosted on Inria is for restricted use within our institution. @mdiazmel can you confirm?

It's in the works though, see #75

**Comment by mdiazmel** [comment]
Hi @vnckppl ,

Can you tell us how do you plan to use Clinica (`run`, `convert`, `iotools`, anything else)? In fact, the creation of the Docker depends on the use of the software. (E.g., we can provide you a `Dockerfile` quickly if it's only for using the converters).

**Comment by vnckppl** [comment]
Hi @mdiazmel, 
I would use Clinca to run adni-to-bids. The main reason is to rule out that any issues that I am currently experiencing with this pipeline (see this thread: https://github.com/aramis-lab/clinica/issues/394) are not stemming from installation related issues. Thanks!

**Comment by ghisvail** [comment]
> The main reason is to rule out that any issues that I am currently experiencing with this pipeline (see this thread: #394) are not stemming from installation related issues.

You don't really need Docker to get a fresh runtime to test converters with. Just setup a new `conda` environment, install `clinica` and `dcm2niix` in it and you are good to go.

**Comment by mdiazmel** [comment]
I'm not sure that using a Docker container will help you with the mentioned issue.

Anyway, you can try to build a Docker container to run the `adni-to-bids` converter by following these instructions:

1. Put this code inside a file named `Dockerfile`:
```
FROM continuumio/miniconda3

RUN apt-get update && \
    apt-get install -y \
      build-essential 

RUN conda install python==3.7
RUN conda install -c conda-forge dcm2niix
RUN pip install clinica==0.5.0 
```

2. Build the docker image by running `docker build -t clinica_conv .` inside the folder containing the `Dockerfile` (Here, we will name the image `clinica_conv`, but you can use a different name).

3. Once the image is built, you can launch a Docker container based on this image, with Clinica inside:
```
docker run -v /path/to/your/adni/dataset/:/root/tmp/adni/ --rm -it clinica_conv /bin/bash
```
This command will start an interactive container and it will mount the folder containing your Adni data inside the container.

4. In the container, you can use Clinica as described in the documentation, something like this:
```
clinica convert adni-to-bids  /root/tmp/adni//unorganized_data /root/tmp/adni/clinical_data /your/bids/location
```
and the options that you want to add.

Hope these instructions will be helpful.

Best,

**Comment by ghisvail** [comment]
You might need to adjust `RUN pip install clinica==0.5.0` to install clinica in editable mode if you want to test the latest development version. But again, there are no benefits in running Docker for this particular use case (testing converters), and there are definitely drawbacks in terms of more complex calls, plus cleaning the image layers and containers afterwards.

**Comment by ghisvail** [comment]
Duplicate of #75
